src/get-message.js

```diff
@@ -1,42 +1,97 @@
 'use strict';
 
+// The size (kb) to count as a significant bundles size change.
+const threshold = 0.2;
+
+/**
+ * Get the bundle size difference between two bundles in KB.
+ * @param {Object} current - The more recent bundle for comparison.
+ * @param {Object} previous - The bundle which pre-dates the current bundle.
+* @return {String}
+ */
 function getSizeDiff(current, previous) {
 	return ((current - previous) / 1024).toFixed(2);
 }
 
-function getMessage(current, previous, brand) {
+/**
+ * Get a message to describe the bundle size difference between two sets of bundles.
+ * @param {Object} current - The more recent bundle for comparison.
+ * @param {Object} previous - The bundle which pre-dates the current bundle.
+ * @return {String}
+ */
+function getBundleComparisonMessage(current, previous) {
 	const gzipDiff = getSizeDiff(current.sizes.gzip, previous.sizes.gzip);
 	const rawDiff = getSizeDiff(current.sizes.raw, previous.sizes.raw);
 
-	if (Math.abs(rawDiff) < 0.5) {
-		return ''; // no significant chnage, -0.5kb
+	// Use the current brand in the key in case the previous version is unbranded,
+	// but the current version has just added brand support.
+	const key = `${current.language}${current.brand ? `, ${current.brand}` : ''}`;
+
+	if (Math.abs(rawDiff) === 0) {
+		return `${key}: no difference.`;
+	}
+
+	if (Math.abs(rawDiff) < threshold) {
+		return `${key}: no significant difference.`;
 	}
 
-	const key = `${previous.language}${brand ? `, ${brand}` : ''}:`;
 	const raw = `${Math.abs(rawDiff)}kb ${rawDiff < 0 ? 'decrease' : 'increase'}`;
 	const gzip = `(${gzipDiff}kb/gzip)`;
-
-	return [
-		key,
-		raw,
-		gzip
-	].filter(m => m).join(' ');
+	return `${key}: ${raw} ${gzip}`;
 }
 
 module.exports = (fromBundles, toBundles) => {
-	let message = '';
-	toBundles.forEach(current => {
-		const brand = current.brand;
+	// Find a matching bundle from a previous version to compare with the
+	// current bundle. Compare bundles of the same language and brand, or
+	// unbranded versions against the master brand of a branded version.
+	const bundleComparisons = toBundles.map(current => {
 		const previous = fromBundles.find(previous => (
 			previous.language === current.language &&
 			(previous.brand === current.brand || (previous.brand === null && current.brand === 'master'))
 		));
-		if (previous) {
-			const part = getMessage(current, previous, brand);
-			if (part) {
-				message += (message ? '\n' + part : part);
-			}
-		}
+		return {
+			current,
+			previous
+		};
+	}).filter(b => b.previous);
+
+	// Message if no bundle sizes have changes at all.
+	const unequalComparisons = bundleComparisons.filter(c => {
+		const rawDiff = getSizeDiff(c.current.sizes.raw, c.previous.sizes.raw);
+		return Math.abs(rawDiff) !== 0;
+	});
+
+	if (unequalComparisons.length === 0) {
```

Comment:
Since this is using a filter function and checking that all items were removed, we can replace it with a hopefully more semantic and readable function called `every`.

```suggestion
	const noBundleSizeChanges = bundleComparisons.every(c => {
		const rawDiff = getSizeDiff(c.current.sizes.raw, c.previous.sizes.raw);
		return Math.abs(rawDiff) === 0;
	}
	if (noBundleSizeChanges) {
```

Reply:
This is only possible for `noBundleSizeChanges`. The filtered arrays are used in the other cases.